rest-client/src/main/java/org/sagebionetworks/bridge/rest/LoggingInterceptor.java

```diff
@@ -57,7 +57,12 @@ public Response intercept(Chain chain) throws IOException {
             Response newResponse = response.newBuilder()
                     .body(ResponseBody.create(body.contentType(), bodyString.getBytes())).build();
 
-            logger.debug(response.code()+" RESPONSE: "+redactPasswords(bodyString));
+            StringBuilder sb = new StringBuilder();
```

Comment:
Make it visually easier to line up a request with its response in a multi-threaded environment